docs/amazon.aws.s3_bucket_module.rst

```diff
@@ -268,7 +268,7 @@ Parameters
                 <td>
                 </td>
                 <td>
-                        <div>The JSON policy as a string.</div>
+                        <div>The JSON policy as a string. Set to <code>"null"</code> to force the absence of a policy.</div>
```

Comment:
The rst doc files are auto-generated from the module code when we do a release, for future PRs you don't need to change this.

Reply:
Oh, thanks for the heads-up. I should have realized that. I removed the generated code from the PR.